recipe/meta.yaml

```diff
@@ -33,9 +28,11 @@ about:
   home: http://github.com/pyviz/pyct
   license: BSD 3-Clause
   license_family: BSD
-  license_file: LICENSE
-  summary: python package common tasks for users (e.g. copy examples, fetch data, ...)
+  license_file: LICENSE.txt
+  summary: 'Python package common tasks for users (e.g. copy examples, fetch data, ...)'
+  doc_url: https://github.com/pyviz/pyct
+  dev_url: https://github.com/pyviz/pyct
 
 extra:
   recipe-maintainers:
-    - ceball
+    - ocefpaf
```

Comment:
You don't have to remove yourself from there.

PS: as soon as the `pyct-core` package is available I'll restart things here. Once I get some free time I'll send a PR to merge the recipes.